src/script/processTarget/setComponent/setComponent.js

```diff
@@ -1,18 +1,19 @@
-const path = require('path')
 const fs = require('fs')
+const path = require('path')
+const reducers = require('../../redux/reducers')
 
-module.exports = (fileName) => {
+module.exports = fileName => async (dispatch) => {
   const name = path.basename(fileName)
   const absolute = path.resolve(process.env.PWD, fileName)
-  const dir = fs.lstatSync(absolute).isDirectory() ? absolute : path.dirname(absolute)
+  const dir = (await fs.lstatAsync(absolute)).isDirectory() ? absolute : path.dirname(absolute)
```

Comment:
maybe here we will need to handle errors (catch) here (and in other files like `setProject`, `setWebpack`...)
but I think it's not this PR purpose, so we may need to create an issue about it.